bears/general/InvalidLinkBear.py

```diff
@@ -108,13 +115,22 @@ def run(self, filename, file,
         `do_not_ever_open = 'https://api.acme.inc/delete-all-data'` wiping out
         all your data.
 
-        :param timeout:          Request timeout period.
+        :param network_timeout:       A dict containing the url and the timeout
```

Comment:
Missing period + I would add a bit more description, why and when this is useful, and how it does work exactly (extracts hostname from given address, etc.)